api/routes/assessment_routes.py

```diff
@@ -119,9 +119,9 @@ def get_sub_criteria_theme_answers(application_id: str, theme_id: str):
     return map_application_with_sub_criteria_themes(application_id, theme_id)
 
 
-def update_ar_status_to_completed(application_id: str):
-    """ Function updates the status to COMPLETE for the given application_id"""
-    update_status_to_completed(application_id)
+def update_ar_status(application_id: str, status: str):
```

Comment:
Can we try to avoid acronyms, so in this case can we call it `update_assessment_status`?